src/webgpu/api/operation/rendering/depth.spec.ts

```diff
@@ -3,13 +3,211 @@ Test related to depth buffer, depth op, compare func, etc.
 `;
 
 import { makeTestGroup } from '../../../../common/framework/test_group.js';
+import { TypedArrayBufferView } from '../../../../common/util/util.js';
 import { kDepthStencilFormats, kTextureFormatInfo } from '../../../capability_info.js';
 import { GPUTest } from '../../../gpu_test.js';
 
+class DepthTest extends GPUTest {
+  checkStencilCompareFunction(
+    compareFunction: GPUCompareFunction,
+    stencilColor: number[],
+    stencilRefValue: number,
+    stencilColorExpected: boolean
+  ) {
+    const depthSpencilFormat: GPUTextureFormat = 'depth24plus-stencil8';
+    const baseColor = [255, 255, 255, 255];
+
+    const baseStencilState = {
+      compare: 'always' as GPUCompareFunction,
+      failOp: 'keep' as GPUStencilOperation,
+      depthFailOp: 'keep' as GPUStencilOperation,
+      passOp: 'replace' as GPUStencilOperation,
+    };
```

Comment:
µNit: a single `as const` at the end here should work:
```ts
    const baseStencilState = {
      compare: 'always',
      failOp: 'keep',
      depthFailOp: 'keep',
      passOp: 'replace',
    } as const;
```

same on the others below

Also I don't think `depthFailOp` is relevant to this test so you could just omit it (it's optional).

Reply:
Thanks for informing me of the use. Fixed and remove `depthFailOp`.